### ISSUE TITLE
[Enhancement] Option to get last item rather than first (in multi-item field value)

### ISSUE THREAD

**Issue by nicolas-raoul** [Pascalco/harvesttemplates#149]
I am trying to get birthplaces from the Japanese Wikipedia so I wrote https://tools.wmflabs.org/pltools/harvesttemplates/?siteid=ja&project=wikipedia&namespace=0&property=19&template=Infobox%20%E4%BA%BA%E7%89%A9&parameter=%E7%94%9F%E8%AA%95%E5%9C%B0&parameter=&addprefix=&removeprefix=&category=&depth=0&set=1&offset=0&limit=1000000&wikisyntax=0&

Problem: In that parameter, the important item comes last.

Example from https://ja.wikipedia.org/wiki/%E4%BA%94%E5%91%B3%E4%B8%80%E7%94%B7 :
```
|生誕地	=  [[長野県]][[諏訪郡]][[富士見町]]
```
In other words, the birth place is Nagano prefecture > Suwa district > Fujimi village

The birthplace in Wikidata should be set to `Fujimi village`, but unfortunately with HarvestTemplates it becomes set to `Nagano prefecture`, the least precise.

Could we have an option to get the latest item rather than the first?
Or is this option already available, maybe?
Thanks a lot!
